### DISCUSSION TITLE
Permettre de télédéclarer les diagnostics de ces cantines en quelques clics

### DISCUSSION THREAD

**Discussion by gauthierandre** [betagouv/ma-cantine#1834] [Retours utilisateur·rice]
Il serait utile d'envisager un bouton qui permet de **télédéclarer l'ensemble de ses sites en un clic**, au lieu de devoir cliquer sur chacune des cartes. En effet, la manipulation peut être longue et décourageante quand on gère plusieurs dizaines de sites, avec des diagnostics alimentés via l'outil d'import. 

Le bouton pourrait apparaitre dans le profil utilisateur / sous la liste des cantines / en suggestion lorsqu'on télé-déclare une première cantine / voire en option dans l'outil d'import (tele declaration : yes / no).

Qu'en penses-vous ? @alemangui @hfroot @JenniferStephan @valeriemerle

**Comment by alemangui** [comment]
Bonne idée, après je me demande s'il faut envisager une façon pour montrer le récapitulatif de chaque cantine avant de télédéclarer :thinking:

**Reply by gauthierandre** [reply]
Ah oui, effectivement. Il y a cette option à prendre en compte. Je n'ai pas spécialement d'idée pour permettre de consulter le diagnostic avant validation.
